src/components/Toast/Toast.tsx

```diff
@@ -45,21 +45,20 @@ export const Toast = ({
 }: Props) => {
   const componentClassName = clsx(
     className,
-    styles["toast"],
-    variant === "success" && styles["toast--success"],
-    variant === "error" && styles["toast--error"],
+    styles['toast'],
+    variant === 'success' && styles['toast--success'],
+    variant === 'error' && styles['toast--error'],
   );
   return (
     <div className={componentClassName} {...other}>
-      <div className={styles["toast__content"]}>
+      <div className={styles['toast__content']}>
         <Icon
-          color="var(--icon-color)"
-          name={variant === "success" ? "check-circle" : "cancel"}
+          name={variant === 'success' ? 'check-circle' : 'warning'}
           purpose="informative"
-          size="1.25rem"
+          size="1.5rem"
```

Comment:
I wonder why this was changed.. 🤔

Reply:
Now it's the same size as in `Banner` but they're different colors. I'll just leave it for now because we will probably have to make some other small design changes to these components as part of the core component work.

Reply:
oh funky... thanks for noticing that 🧐